docs/source/index.rst

```diff
@@ -88,6 +88,51 @@ show an example of how to annotate variants with the genes that they overlap.
 
     w.close(); vcf.close()
 
+Here is an example where we filter some calls from records in a VCF. This demonstrates
```

Comment:
This is too long for the main index page. Can you add another page and link from here?